src/FileSearching.java

```diff
@@ -0,0 +1,28 @@
+import java.io.File;
+import java.util.Scanner;
+
+class FileSearching {           ///////////////checking of file avaibility
+   protected FileSearching(String file)  {
+        if(file.endsWith("txt") || file.endsWith("json")){
+        FileChecking(file);
+        }
+    }
+
+    private static void FileChecking(String filepath){
+        try {
+            String sn="";
+            File f = new File(filepath);
+            Scanner s= new Scanner(f);
+            while( s.hasNextLine()){
+                sn=sn+s.nextLine();
```

Comment:
try to use StringBuffer

Reply:
Now I changed my code and using Path class to read file .